server/server.js

```diff
@@ -84,6 +93,21 @@ app.get("/api/customers/:customer_id/orders", (req, res) => {
   //   }
   // ];
 });
+
```

Comment:
The reason would be that it would avoid unnecessary DB calls, potential errors by blindly trusting the user input, and by making things explicit we make the code self-documenting.